### DISCUSSION TITLE
Writing dusk tests for filament forms

### DISCUSSION THREAD

**Discussion by gsusanj** [filamentphp/filament#8324] [Help]
Hi there, we have been trying to write some dusk browser tests with the filament forms, but we are having issues with the select component.

Anyone got any recommendation or resource how to do these tests?

**Comment by leandrocfe** [comment]
What issue?

You can use extraInputAttributes to define [dusk-selectors](https://laravel.com/docs/10.x/dusk#dusk-selectors)
```php
Select::make('my_select')
    ->extraInputAttributes(['dusk' => 'my-select'])
```
